packages/components/psammead-radio-schedule/src/testHelpers/helper.jsx

```diff
@@ -6,6 +6,41 @@ import React from 'react';
 import ProgramCard from '../ProgramCard';
 import RadioSchedule from '../index';
 
+// New locales
+import '@bbc/psammead-locales/moment/am';
```

Comment:
Should we be importing locales to be available for all stories, instead of just in the radio schedule test helper?

Reply:
@tochwill Good idea, I intend to do that along with some other cleanups in a [separate PR ](https://github.com/bbc/psammead/pull/3436).